lib/streamlit/runtime/secrets.py

```diff
@@ -227,6 +227,20 @@ def _parse(self, print_exceptions: bool) -> Mapping[str, Any]:
 
             return self._secrets
 
+    def to_dict(self) -> dict[str, Any]:
+        """Converts the secrets store into a nested dictionary, where nested AttrDict objects are also converted into dictionaries."""
+        secrets = self._parse(True)
+        return self._convert_to_dict(secrets)
+
+    @staticmethod
+    def _convert_to_dict(obj: Any) -> Any:
```

Comment:
I think it is better to just put this method outside of the class (somewhere on top of this module) since I believe there isn't a need why this needs to be a static class method.